### PR TITLE
Adapting some linux entries

### PR DESCRIPTION
**Pull request by temcom**

### CODE REVIEW

File: BUILD_LINUX.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Please read the [general build guide](BUILD.md) for information on dependencies required for all platforms. Only Linux specific instructions are found in this file.
 
-## Ubuntu 16.04/18.04 specific build guide
+## Ubuntu 16.04/18.04/Mint 19 Tara specific build guide
```
[the remark this answers is not in the document]
Comment (temcom) [reply BUILD_LINUX.md]: No problem with that, I reflected my success on this distribution, only.
I will doing the work on the Ubuntu in future, I`ve seen too much problems on the hifidelity linux interface front in the past:-)
Thank you for your fast response 
For me it is a starting point - thank you so much 4 your work

[the remark this answers is not in the document]
Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: We're very happy to have you!
When we get a new commit from you, I'll resolve this change request and approve it. Alternately, I can make the change if you prefer.

[the remark this answers is not in the document]
Comment (MarcusLlewellyn) [reply BUILD_LINUX.md]: I've gone ahead and made the edit I requested.

**Comment by kasenvr** [comment]
The only other issue I can think of is the part regarding tags but we can iron that out later.

**Pull request merged**